scripts/process-docfx-output.js

```
@@ -0,0 +1,565 @@
+    content += `**Implements:** ${implementsList}\n\n`;
+  }
+
+  // Process children from the class
+  if (classItem.children && classItem.children.length > 0) {
+    const groupedChildren = groupChildren(classItem.children, data);
+
+    // Add constructors
+    if (groupedChildren.Constructor && groupedChildren.Constructor.length > 0) {
+      content += "## Constructors\n\n";
+      groupedChildren.Constructor.forEach((constructor) => {
+        content += generateMethodComponent(constructor, false);
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)